Relax minimum zstd for 2021Q2

-- Could NOT find zstd: Found unsuitable version "1.4.8", but required is at least "1.5" (found /usr/local/lib/libzstd.so)
-- Packages zstd/1.5.0 not found!
-- Downloading conan.cmake from https://github.com/conan-io/cmake-conan
CMake Error at CMakeLists.txt:358 (conan_check):
  Unknown CMake command "conan_check".

--- CMakeLists.txt.orig	2021-05-26 22:19:02 UTC
+++ CMakeLists.txt
@@ -174,7 +174,7 @@ macro(yuzu_find_packages)
         "lz4               1.8         lz4/1.9.2"
         "nlohmann_json     3.8         nlohmann_json/3.8.0"
         "ZLIB              1.2         zlib/1.2.11"
-        "zstd              1.5         zstd/1.5.0"
+        "zstd              1.4         zstd/1.4.8"
     # can't use opus until AVX check is fixed: https://github.com/yuzu-emu/yuzu/pull/4068
         #"opus              1.3         opus/1.3.1"
     )
